app/src/main/java/com/cornellappdev/android/eatery/MainActivity.java

```diff
@@ -70,6 +74,7 @@
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
+    Fabric.with(this, new Crashlytics());
```

Comment:
Line needs to be deleted, it is already instantiated in EateryApplication.java

Reply:
done